src/sidebar/reducers/selection.js

```diff
@@ -132,6 +133,17 @@ var update = {
     };
   },
 
+  ADD_ANNOTATIONS(state, action) {
+    var counts = tabs.counts(action.annotations);
+    // If there are no annotations at all, ADD_ANNOTATIONS will not be called.
+    var haveOnlyPageNotes = counts.notes === action.annotations.length;
+    // If this is the init phase and there are only page notes, select the page notes tab.
+    if (state.annotations.length === 0 && haveOnlyPageNotes){
+      return {selectedTab: uiConstants.TAB_NOTES};
+    }
+    return {selectedTab: state.selectedTab};
```

Comment:
You can actually `return null` or `return {}` here if you don't want to make any changes. The returned changes are merged into a copy of the existing state.